docs/processes/release/tasks/publish/release.md

```diff
@@ -73,10 +73,21 @@ Here you should use the release tag.
      the promotion to Maven Central.
    * The unified Java documentation, (`icu4j-76.1-fulljavadoc.jar`) \
      **WARNING:** this is also the one to be published (unpacked) for web access
+   * Make sure to set the proper parameters:
+     * **Branch:** the branch prepared for release, for example maint/maint-77
+     * **Run the tests:** checked (default)
+     * **Deploy to Maven Central:** check (unchecked by default), if you are ready
+       for a real deployment and not doing just a sanity check
+     * **Release tag to upload to:** should be the GitHub draft release prepared
+       in a previous BRS step.
 
 1. **Release - Create checksums and GPG sign** (`release-check-sign.yml`) \
    THIS SHOULD BE THE LAST ACTION YOU RUN. \
    After all the artifacts from the previous steps are posted to the release. \
    The action will download all the artifacts from release,
    create checksum files (`SHASUM512.txt` and `*.md5`),
    and digital signature files (`*.asc`)
+
+1. **Login to Sonatype, sanity check, and approve** \
+  See [../release-build.md](../release-build.md) \
+  **TODO: should we merge the content of that document here and remove it?**
```

Comment:
My answer to the question is yes -- refactor the content of `release-build.md` and then remove the file. After the current changes by this PR to that file, there won't be much left to justify keeping that a separate file since what will remain will be exclusively about ICU4J release build artifact deployment.